src/main/kotlin/com/coder/gateway/cli/CoderCLIManager.kt

```diff
@@ -280,14 +279,26 @@ class CoderCLIManager(
                 transform = {
                     """
                     Host ${getHostName(deploymentURL, it)}
-                      ProxyCommand ${proxyArgs.joinToString(" ")} $it
+                      ProxyCommand CODER_SSH_USAGE_APP=jetbrains ${proxyArgs.joinToString(" ")} $it
```

Comment:
Thinking about this more, I think we might have to use a flag after all because iirc this syntax does not work for Windows.